helix-rest/src/main/java/org/apache/helix/rest/server/ServerContext.java

```diff
@@ -118,6 +120,29 @@ public HelixDataAccessor getDataAccssor(String clusterName) {
     }
   }
 
+  public ZkBaseDataAccessor<byte[]> getByteArrayBaseDataAccessor() {
+    if (_byteArrayBaseDataAccessor == null) {
+      synchronized (this) {
+        if (_byteArrayBaseDataAccessor == null) {
+          _byteArrayBaseDataAccessor = new ZkBaseDataAccessor<>(_zkAddr, new ZkSerializer() {
+            @Override
+            public byte[] serialize(Object o)
+                throws ZkMarshallingError {
+              return (byte[]) o;
```

Comment:
Since we don't support this yet, let's throw an unsupported exception?

Reply:
done.